src/pal/inc/unixasmmacrosarm64.inc

```diff
@@ -31,8 +31,8 @@ C_FUNC(\Name):
 .endm
 
 .macro LEAF_END_MARKED Name, Section
-        .global C_FUNC(\Name\()_End)
 C_FUNC(\Name\()_End):
+        .global C_FUNC(\Name\()_End)
```

Comment:
At the time I was trying to remove cosmetic difference between amd64 and arm64.  I'll take a look.